crowdnotifier-sdk/src/main/java/org/crowdnotifier/android/sdk/CrowdNotifier.java

```diff
@@ -55,9 +57,8 @@ public static boolean updateCheckIn(long id, long arrivalTime, long departureTim
 			for (ExposureEvent match : matches) {
 				if (match.getStartTime() <= problematicEventInfo.getEndTimestamp() &&
 						match.getEndTime() >= problematicEventInfo.getStartTimestamp()) {
-					ExposureEvent newExposureEvent = new ExposureEvent(match.getId(),
-							Math.max(match.getStartTime(), problematicEventInfo.getStartTimestamp()),
-							Math.min(match.getEndTime(), problematicEventInfo.getEndTimestamp()), match.getMessage());
+					ExposureEvent newExposureEvent =
```

Comment:
why do we need to create a new instance of a ExposureEvent here? could we not just use match directly?

Reply:
Good point. Since we clipped the arrival and departure times of the match by the start and end time of the exposure in the previous version, we had to create a new object. But since we decided not to do this clipping anymore, we can directly use match.